packages/dlib/meta.yaml

```
@@ -0,0 +1,18 @@
+package:
+  name: dlib
+  version: 19.22.1
+
+source:
+  url: https://files.pythonhosted.org/packages/f0/a2/ba6163c09fb427990180afd8d625bcecc5555af699c253193c35ffd48c4f/dlib-19.22.1.tar.gz
+  sha256: 77e943b40cc53e377d7fc683552d2337937f1111c46d7753105f594c3958163c
+
+build:
+  cxxflags: -std=c++11
+  cflags: -std=c++11
+  setupflags: |
```

Maybe,

```suggestion
  setup_install_args: |
```
?

I find it fairly unreadable otherwise without spaces.